### PR TITLE
Michellemerrill patch 2

### PR DESCRIPTION
**Pull request by nine03**
.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,6 +6,7 @@ In this repository, you can find the [official GitHub public feedback discussion
 - [GitHub Discussions](https://github.blog/2020-05-06-new-from-satellite-2020-github-codespaces-github-discussions-securing-code-in-private-repositories-and-more/#discussions) - [:speaking_head: Discussions Feedback](https://github.com/github/feedback/discussions/categories/discussions-feedback)
 - [GitHub Issues](https://github.com/features/issues) - [:octopus: Issues Feedback](https://github.com/github/feedback/discussions/categories/issues-feedback)
 - [GitHub Mobile](https://github.com/mobile) - [:iphone: Mobile Feedback](https://github.com/github/feedback/discussions/categories/mobile-feedback)
+- [GitHub Releases](https://docs.github.com/en/repositories/releasing-projects-on-github/automatically-generated-release-notes) - [❇️ Releases Feedback](https://github.com/github/feedback/discussions/categories/releases-feedback)
```
Comment (shafeeqshebu) [inline README.md:9]: 👍

**Review by shafeeqshebu — COMMENTED** [review]
Exactly

**Review by shafeeqshebu — COMMENTED** [review]
Nice

File: README.md (hunk 1 above)
Comment (zdsxz) [reply README.md:9]: شكرا

**Pull request closed without merging**